canonical-web-and-design/react-components
# Fix storybook docs, and simplify/correct config.

## Done
* Remove unnecessary config for v6.
* Fix glob syntax as per the  v5-6 [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-53x-to-60x). 
* Upgrade all storybook deps to be in lockstep (addons and storyboard/react must all be the same version, or things break)
* Move stylelint to dev-deps.

## QA
* Run storybook (`yarn start`)
* Both canvas and docs tabs should render correctly without errors.
